internal/r8/map_fetcher.go

```diff
@@ -0,0 +1,100 @@
+package r8
+
+import (
+	"bytes"
+	"compress/zlib"
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"net/url"
+
+	"github.com/elastic/apm-server/internal/elasticsearch"
+	"github.com/elastic/apm-server/internal/logs"
+	"github.com/elastic/elastic-agent-libs/logp"
+	"github.com/elastic/go-elasticsearch/v8/esapi"
+)
+
+const mapIndex = ".apm-android-map"
+
+var errFetcherUnvailable = errors.New("fetcher unavailable")
```

Comment:
```suggestion
var errFetcherUnavailable = errors.New("fetcher unavailable")
```

Reply:
Than you, I'll add the changes locally to get the IDE to replace the name where this is referenced.